CHANGELOG.md

```diff
@@ -680,9 +691,22 @@ Other minor changes
   ^↔→                : Extensionality _ _ → Fin (n ^ m) ↔ (Fin m → Fin n)
   ```
 
+* Added new functions in `Data.Integer.Base`:
+  ```
+  _^_ : ℤ → ℕ → ℤ
+  ```
+
 * Added new proofs in `Data.Integer.Properties`:
   ```agda
   sign-cong′ : s₁ ◃ n₁ ≡ s₂ ◃ n₂ → s₁ ≡ s₂ ⊎ (n₁ ≡ 0 × n₂ ≡ 0)
+  ¬[1≡0] : ¬ 1ℤ ≡ 0ℤ
```

Comment:
This entry should be removed and entries for `∣-∣-≤`, `∣⊖∣-≤` and `≤-⊖` added.

Reply:
Working on that now.

Reply:
Good to go now, I think.